components/CollectionCard.js

```diff
@@ -61,9 +91,44 @@ export default function CollectionCard({ flashcards, collection }) {
           <p>Correct Cards: {correctFlashcards.length}</p>
         </CollectionCardArticle>
       </StyledCardLink>
+
+      <DeleteButton
+        onClick={(event) => {
+          toggleConfirmation();
+        }}
+      >
+        delete
+      </DeleteButton>
```

Comment:
You don't make use of the `event` here, so there is no real need for a callback function here. 
And being explicit about the button type is always a good idea:

```suggestion
      <DeleteButton
        onClick={toggleConfirmation}
        type="button"
      >
        delete
      </DeleteButton>
```